### ISSUE TITLE
Display list of winners in Leaderboard and end game

### ISSUE THREAD

**Issue by nitro2** [dnh-bot/dnh-werewolf-bot#125]
Sample Result in Leaderboard:
```
Kết quả trò chơi
Trò chơi đã bắt đầu lúc 03:32:21 ngày 31-01-2024.
Số ngày đã trải qua
5
🏆 Phe chiến thắng
Villager
 Player chiến thắng
@B, @C, @D, @E
📝 Danh sách role
@A là Werewolf
@B là Seer
@C là Guard
@D là Zombie
@E là Lycan
@F là Fox
```

If Cupid wins, the list includes the couple and cupid
If Villager wins, the list also includes Cupid but not Betrayer
If Fox wins, the list only includes Fox
If Werewolf wins, the list includes Werewolf and Betrayer

**Comment by huybt22** [comment]
I think it should be a simple emoji for winners like this, that's also avoid redundant information and player mentions

`
Kết quả trò chơi
Trò chơi đã bắt đầu lúc 03:32:21 ngày 31-01-2024.
Số ngày đã trải qua
5
🏆 Phe chiến thắng
Werewolf

📝 Danh sách role
@A là Werewolf 🎉
@B là Seer
@C là Guard
@D là Superwolf 🎉
@E là Lycan
@F là Fox
`

**Comment by nitro2** [comment]
Agree

**Comment by huybt22** [comment]
Implemented at #129 , please have a look.
